unsupported word formatting (#7807)
MicrosoftDocs/microsoft-365-docs: microsoft-365/contentunderstanding/content-assembly.md

      - Autofill placeholder values if able to uniquely identify a record for placeholders associated with the same list.

> [!NOTE]

> > - Currently, only Microsoft Word documents (.docx extension) are supported for creating a template. Before uploading the document, ensure that the Word document doesn't have **Track changes** enabled or comments. If your document contains text placeholders for images, ensure that they are not text-wrapped. We do not support **Content Controls** in word at the moment. If you want to create a template from a word docuemnt with content controls, please remove them before creating a modern template.

Reviewer: ```suggestion
> - Currently, only Microsoft Word documents (.docx extension) are supported for creating a template. Before uploading the document, ensure that the Word document doesn't have **Track changes** enabled or comments. If your document contains text placeholders for images, ensure that they are not text-wrapped.
> - Currently, we do not support **content controls** in Word. If you want to create a template from a Word document with content controls, you will need to remove them before creating a template.
```
